src/ui/screens/Parachains/ParathreadsScreen.tsx

```diff
@@ -65,7 +65,7 @@ type ParathreadItemProps = {
 function ParathreadItem({id, leases, leasePeriod}: ParathreadItemProps) {
   const {data: parathreadInfo} = useParathreadInfo(id);
   const parathreadData = useParathreadData(id);
-  const {data: manager} = useAccountIdentityInfo(parathreadInfo?.manager?.toString());
+  const {data: manager} = useAccount(parathreadInfo?.manager?.toString());
```

Comment:
Can you add a TODO here to remove this? The manager will come from the api in the `Parathread` object